.t9n.yml

```diff
@@ -17,5 +18,5 @@ exclude_files:
   - node_modules
   - vendor
   - tools
-  - mails
+  - mails/[a-zA-Z][a-zA-Z]/
```

Comment:
```suggestion
  - mails/[a-zA-Z]{2}/
```
If there is only two characters

Reply:
I tried it again just in case, and it works! (I didn't have the las `/` at the time which is why it didn't work)